source/upgrade/important-upgrade-notes.rst

```diff
@@ -21,6 +21,10 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v6.2                                               | Channel results in the channel switcher will include channels from other teams as well as private channels. For users using Bleve or Elasticsearch for           |
```

Comment:
```suggestion
| v6.2                                               | Channel results in the channel menu will include channels from other teams as well as private channels. Customers using Bleve or Elasticsearch for           |
```

Reply:
Recommend moving away from "switch" and "switcher" terminology, as per Marketing.